crossfit/backend/cudf/array.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
         def concatenate(self, series_list, *, axis=None):
             return cudf.concat(series_list, axis=axis or 0)
 
-    np_backend_dispatch.register((cudf.Series, cudf.GenericIndex))(CudfBackend())
+    np_backend_dispatch.register((cudf.Series, cudf.Index))(CudfBackend())
```

Comment:
Unsure if this is the right fix. CC: @rjzamora

Reply:
Yes, this looks correct to me. My understanding is that all "real" index objects inherit from `Index` now.

I suppose dask-cudf uses [BaseIndex](https://github.com/rapidsai/cudf/blob/34dea6fe40fc20966b48257853865111df4a687f/python/dask_cudf/dask_cudf/backends.py#L52), but that dispatch was added a long time ago (and BaseIndex is probably unnecessary).